include/eosio/vm/debug_visitor.hpp

```diff
@@ -5,14 +5,14 @@
 
 #include <iostream>
 
-#define DBG_VISIT(name, code)                                                                                          \
+#define DBG_VISIT(name, code, fmt)                                                                                          \
    void operator()(EOS_VM_OPCODE_T(name)& op) {                                                                        \
```

Comment:
Could be `const` reference if it wasn't for the fact that for some reason the overloads that take `block_t`, `loop_t`, and `if_t` take them by non-const reference (though I don't think they should be).